initial-promises/node-server/promises.cf

```diff
@@ -437,6 +437,12 @@ bundle agent garbage_collection
         file_select => days_old("30"),
         depth_search => recurse("inf");
 
+    enterprise_edition::
```

Comment:
ha, actually, $(sys.workdir)/promise_summary.log also need to be rotated
and on enteprise only, files:
        "$(sys.workdir)/cf_notkept.log",
        "$(sys.workdir)/cf_repair.log",
        "$(sys.workdir)/state/cf_value.log",
        "$(sys.workdir)/outputs/dc-scripts.log",
        "$(sys.workdir)/state/promise_log.jsonl",
        "$(sys.workdir)/state/classes.jsonl",